validate.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/bash
+
+RUNS_PER_THREAD=100000
+MAX_PTHREADS=$(nproc)
+BIN_PATH="./build/bin"
+RES_FILE="res.txt"
+BUILD_LOG="build.log"
+STEP_SELETCTED=0
+STEP_THREADS=1
+FIBERS=""
+NUMA_NODES=""
```

Comment:
Same here, no default value. This means that a `""` is a valid value for tests, right?

Reply:
In case this variable is empty, an empty string is passed in the arguments of the benchmark.
In case that the user runs validate.sh with explicit extra arguments, this string is not empty and a non empty string is passed in the arguments of the benchmark.